resources/js/beatmap-discussions/discussion.tsx

```diff
@@ -119,6 +119,9 @@ export class Discussion extends React.Component<Props> {
         className={`${topClasses} js-beatmap-discussion-jump`}
         data-id={this.props.discussion.id}
         onClick={this.handleSetHighlight}
+        style={{
+          '--discussion-colour': `var(--beatmapset-discussion-colour--${kebabCase(this.props.discussion.message_type)})`,
```

Comment:
I think it's fine to use the value as is. It looks a bit weird but otherwise simplifies the handling of this.

...and also there are a few mixups in some places regarding casing of this thing